ipc/provider/src/checkpoint.rs

```diff
@@ -199,18 +199,22 @@ impl<T: BottomUpCheckpointRelayer + Send + Sync + 'static> BottomUpCheckpointMan
                     .await?;
                 log::debug!("bottom up bundle: {bundle:?}");
 
-                todo!("implement submit checkpoint")
```

Comment:
This change look unrelated from the fix. Just to be clear this is called by the relayer, right?

Reply:
yeah, it's a minor thing, should have enabled, but too small for another PR.